test/packages/good/elasticsearch/transform/good_example_abc_1/dest_index_template.json

```diff
@@ -0,0 +1,11 @@
+{
+  "template": {
+    "mappings": {
+      "properties": {
+        "@timestamp": {
```

Comment:
Would it be an option to build the mapping from a `fields` directory? Mappings definition would be consistent with other fields definitions in packages, and we could reuse other features we have for static validation.

Reply:
+1 on defining fields the same way as we do for data streams. Also, do we have any use cases where we wouldn't want to the fields defined already? If not, I think we should make it required anyways.

Reply:
After adding the `fields` directory, is this file needed now?